src/main/java/org/circle8/dao/PuntoReciclajeDao.java

```diff
@@ -137,7 +179,21 @@ private PreparedStatement createSelectForList(Transaction t, PuntoReciclajeFilte
 
 		var p = t.prepareStatement(b.toString());
 		for (int i = 0; i < parameters.size(); i++)
-			p.setObject(i+1, parameters.get(i));
+			p.setObject(i + 1, parameters.get(i));
+
+		return p;
+	}
+
+	private PreparedStatement createSelectForGet(Transaction t, Long id, Long recicladorId) throws PersistenceException, SQLException {
+		var b = new StringBuilder(SELECT_GET);
+		List<Object> parameters = new ArrayList<>();
+
+		parameters.add(id);
+		parameters.add(recicladorId);
+
+		var p = t.prepareStatement(b.toString());
+		for (int i = 0; i < parameters.size(); i++)
+			p.setObject(i + 1, parameters.get(i));
```

Comment:
No tiene sentido armar una lista siempre del mismo tamaño y luego recorrerla con un for. Directamente se puede hacer
``` java
p.setLong(1, id);
p.setLong(2, recicladorId);
```